tsconfig.json

```diff
@@ -27,7 +27,7 @@
     "module": "ES2022",
     "resolveJsonModule": true,
     "lib": [
-      "ES2022",
+      "ES2024",
```

Comment:
do you know if there are / we need polyfills for any important backward compatibility on older browsers?

Reply:
Yes, we do need polyfills for backwards compatibility. I don’t know what your browser policy is and how it is implemented. But I’m happy to investigate.

Reply:
I don't know either what our browser policy is :laughing: 

But we should support somewhat older browsers. We are catering to a diverse, often low-tech user base.
According to user analytics we had people using Chrome Mobile 119, Firefox 133, Edge 131 or Safari 16.0 this year.

Reply:
@geigerzaehler , do we still need to add the polyfill here before this can be merged?

Reply:
agreed. Let's postpone this for now. I'll close the PR to have a up-to-date overview on GitHub but let's keep the branch for reference.